src/components/link/index.stories.mdx

```diff
@@ -0,0 +1,55 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { ArrowBackIcon, ExternalLinkIcon } from '@chakra-ui/icons';
+
+import Link from './index.tsx';
+
+<Meta title="Components/Link" component={Link} />
+
+# Link
+
+## Overview
+
+<Canvas>
+  <Story name="Overview">
+    <Link href="https://autoscout24.ch">{'AutoScout24 Homepage'}</Link>
+  </Story>
+</Canvas>
+
```

Comment:
```suggestion
## Left Icon
```

Then in the DOCS we can see the subtitle like you did on the button variations. Thanks!